api/CSSAnimation.json

```diff
@@ -29,10 +29,10 @@
             "version_added": null
           },
           "safari": {
-            "version_added": false
+            "version_added": "13.1"
```

Comment:
Mentioned in https://developer.apple.com/documentation/safari-release-notes/safari-13_1-release_notes so this seems right.